spec.md

```diff
@@ -132,18 +132,18 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| id | string | The id of the Oauth client that the dashboard will use. |
-| secret | string | A secret for the dashboard client |
+| id | string | The id of the Oauth client that the dashboard will use. If present, MUST be a non-empty string. |
+| secret | string | A secret for the dashboard client. If present, MUST be a non-empty string. |
 | redirect_uri | string | A URI for the service dashboard. Validated by the OAuth token server when the dashboard requests a token. |
 
 
 ##### Plan Object
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| id* | string | An identifier used to correlate this plan in future requests to the broker. This MUST be globally unique within a platform marketplace. Using a GUID is RECOMMENDED. |
-| name* | string | The CLI-friendly name of the plan. MUST be unique within the service. All lowercase, no spaces. |
-| description* | string | A short description of the plan. |
+| id* | string | An identifier used to correlate this plan in future requests to the broker. This MUST be globally unique within a platform marketplace. MUST be a non-empty string. Using a GUID is RECOMMENDED. |
+| name* | string | The CLI-friendly name of the plan. MUST be unique within the service. MUST be a non-emptry string containing only lowercase alphanumeric characters, no spaces. |
```

Comment:
updated.  The wishy-washy-ness of "All lowercase, no spaces" not being normative makes me twitch a little :-)  but we can deal with that later.  For now at least this makes us more consistent with other spots in the spec.